packages/cloud_firestore/cloud_firestore_web/android/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'cloud_firestore_web'
```

Comment:
I think this should be the same as whatever "package" you have in the AndroidManifest, so: `'io.flutter.plugins.cloud_firestore_web'`?

Reply:
I've taken [this](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_auth/firebase_auth_web/android/settings.gradle) as an example. AFAIK, this should reference the same name as the gradle module generated which would be `cloud_firestore_web`.